hawkbit-security-integration/src/main/java/org/eclipse/hawkbit/security/AbstractControllerAuthenticationFilter.java

```diff
@@ -47,10 +47,10 @@ public boolean isEnable(final TenantSecurityToken secruityToken) {
     }
 
     @Override
-    public abstract HeaderAuthentication getPreAuthenticatedPrincipal(TenantSecurityToken secruityToken);
+    public abstract Object getPreAuthenticatedPrincipal(TenantSecurityToken secruityToken);
```

Comment:
Remove this method. It is already declared by the interface `PreAuthentificationFilter`. This is an abstract class, so there is no need to implement/declare the method as `abstract` here again.